src/EditorFeatures/Test2/Rename/RenameEngineTests.CSharpConflicts.vb

```diff
@@ -2963,7 +2964,7 @@ class C
 {
     void F(int [|$$z|])
     {
-        string x = nameof({|ref:zoo|});
```

Comment:
They exercise the change (and match the repro in the bug). I originally wrote this test this way and that's how I found this bug originally. I can make the test separate if needed,